cbbr_build/sql/apply_corrections.sql

```diff
@@ -0,0 +1,47 @@
+-- Create multi geometry if the records has the same unique_id into a CTE table 
+-- and rename the geometry column from wkt to _multipoint_geom to highlight the change in geometry type
+
+-- Create update points corrections table
+
```

Comment:
I like the comments but also like the sort of header comments below of `--- Try Lines`

would be helpful to add `-- Try Points` here and repeat the `-- Create` and `-- Update` comments in the lines and polygon sections

Reply:
fixed